web/src/components/featuresSection.js

```diff
@@ -42,11 +42,11 @@ export default function FeaturesSection({ images }) {
     return (
         <StyledSection id="why-hubble">
             <StyledHeading>{content.heading}</StyledHeading>
-            {content.features.map((feature, index) => {
+            {content.features.map((feature, i) => {
                 return (
                     <Feature
                         content={feature}
-                        image={images[index]}
+                        image={images[i]}
```

Comment:
Do we still need this (should be in sanity now)? If not let's remove.

Reply:
@asasmith an old component from OldIndex.js

Reply:
same as above

Reply:
@asasmith I changed this one back